extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -903,10 +905,10 @@ export async function newDeployment(
       );
       return;
     }
-    selectedConfigDetails.title = state.data.title;
+    selectedConfigDetails.configuration.title = state.data.title;
     const createResponse = await api.configurations.createOrUpdate(
       configName,
-      selectedConfigDetails,
+      selectedConfigDetails.configuration,
```

Comment:
With the rename of `configDetails` to `inspectionDetails`, should we also rename `selectedConfigDetails` to `selectedInspectionDetails`?